retesteth/ethObjects/stateTest/scheme_transaction.h

```diff
@@ -74,9 +74,20 @@ class scheme_transaction : public object
             SignatureStruct sig;
             getSignedRLP(&sig);
             newData.removeKey("secretKey");
+
+            std::string r = toCompactHexPrefixed(sig.r);
+            if (r.length() == 2) {
+                r = "0x00";
+            }
```

Comment:
I think the same can be achieved via `toCompactHexPrefixed(sig.r, 1)`, as `toCompactHexPrefixed` takes a minimum byte length as second parameter.

Reply:
ah, that'd be perfect.